graduation-project-tips.md

```diff
@@ -0,0 +1,93 @@
+# Ten Things they don't tell you about your graduation project
+
+In this document, I try to list the most important tips and tricks (in my opinion) in the hopes of helping future CCEians by sharing my experience with the graduation project and how to get the most out of it (hopefully).
+
+## Tip #1: Team Comes First :two_men_holding_hands:
+
+The team is the **most important factor** of any project, a team determines how the project starts, how the project ends and pretty much everything in between. 
+You must be comfortable to be yourself with your team and feel among friends, if this condition is not achieved, there will be fights and fights is not good for work.
+
+This is based on true events, it happened in my batch, in previous batches and will happen sadly in future batches (unless everyone reads/applies this tip hopefully).
+So, number one step for me is to gather the team of friends/superheroes you want to roll with. Those who you're ready to partner up with for a one-year contract project. 
+
+## Tip #2: Brainstorming Ideas :bulb:
+
+So, you now have a team that is ready to take on this challenge. **What next?**  
+Start of with couple of meetings, to brainstorm ideas for the project. I always get the question of how to find ideas, and you must understand there is no steps or map you follow for that question. 
+Instead, what I will do is list you some questions that by answering them yourselves, you will have a more clear view of where your idea is going.
+
+- What is the focus/direction of the Project:question:  Is it a research-based project or is it product-based project:question:  
+
+- What techanical fields of study your project is tackling:question: Is it an embedded system project or a web/mobile based project:question:
+
+- What are the hot topics in that field of study we're tackling:question: What problems are related to that field in general:question:  
+
+- What technology stacks/fields your team collectively have of knowledge of:question:  What new techology stacks/fields are you required to learn:question:
+
+Try to answer those questions, they will narrow down your search space at least. You can lookup startup ideas that have previously explored that domain before and find something similar interesting to you.
+Also keep in mind, your idea need not to be a fresh idea. You can choose to re-create an already existing solution in a better fashion/ with new fancy features than those done in the field (Consider Careem company's success while Uber was already there).
+
+As a final note here, bring up more than one idea (one at least as backup plan) in case professors don't find this idea attractive enough for them, you should have other options in your bag.
```

Comment:
Might be worth mentioning some professors expect strong proposals and clear ideas from the get go and other are willing to brainstorm together.

Reply:
Added at the end of Tip #3